src/frontend/js/api/joanie/gen/models/PaginatedAddressList.ts

```diff
@@ -0,0 +1,14 @@
+/* generated using openapi-typescript-codegen -- do no edit */
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+import type { Address } from './Address';
+
+export type PaginatedAddressList = {
```

Comment:
doc: https://drf-spectacular.readthedocs.io/en/latest/faq.html#how-do-i-wrap-my-responses-my-endpoints-are-wrapped-in-a-generic-envelope
todo: try to use spectacular envelope in order to create a PaginatedSerialier
todo: try to use envelope into a custom pagination class https://www.django-rest-framework.org/api-guide/pagination/#custom-pagination-styles